doc/rtd/reference/datasources/nocloud.rst

```diff
@@ -29,10 +47,42 @@ The permitted keys are:
 * ``i`` or ``instance-id``
 * ``s`` or ``seedfrom``
 
-With ``ds=nocloud``, the ``seedfrom`` value must start with ``/`` or
-``file://``.  With ``ds=nocloud-net``, the ``seedfrom`` value must start
-with ``http://`` or ``https://`` and end with a trailing ``/``.
+A valid ``seedfrom`` values consist of:
+
+Filesystem
+----------
+A filesystem path starting with ``/`` or ``file://`` that points to a directory
+containing files: ``user-data``, ``meta-data``, and (optionally)
+``vendor-data``
+
+Http server
+-----------
+An ``http`` or ``https`` url (a trailing ``/`` is required)
+
+
+File formats
+============
+
+These user data and metadata files are required as separate files at the
+same base URL: ::
+
+  /user-data
+  /meta-data
+
+Both files must be present for it to be considered a valid seed ISO.
+
+Basically, ``user-data`` is simply :ref:`user data<user_data_formats>` and
```

Comment:
I know you didn't initially write it, but sentence is better without `Basically`. Also `what you'd find in the EC2 metadata service` isn't very intuitive. It'd be nice to fix here, but doesn't have to be.

Reply:
+1